src/components/Editable/editable.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  TextField,
+  IconButton
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import EditIcon from '@material-ui/icons/Edit';
+
+const useStyles = makeStyles(theme => ({
+  root: {},
+  iconButton: {
+    marginRight: 0
+  }
+}));
+
+function Editable({ logEntry }) {
```

Comment:
super nice!

I've noticed there's one bug though, when adding empty comments. Maybe we should have a validation for that. What do you think?

![image](https://user-images.githubusercontent.com/13377065/123705051-b3383080-d866-11eb-8560-facb927ef6f9.png)